scripts/populate_db.py

```diff
@@ -35,7 +41,7 @@ def run_db_stuff(create_schema, recreate_db, populate_data, drop_db):
         engine = create_engine(CorporaDbConfig().database_uri)
         if database_exists(engine.url):
             print("Database exists, dropping database")
-            drop_database(engine.url)
+            # drop_database(engine.url)
```

Comment:
did you mean to comment this out?

Reply:
@nayib-jose-gloria thanks for catching, I had meant to clean this up after some adhoc testing. It isn't used, per se, but I've reverted it to the previous state. 👍